python_coreml_stable_diffusion/torch2coreml.py

```diff
@@ -271,6 +297,25 @@ def bundle_resources_for_swift_cli(args):
         f.write(requests.get(args.text_encoder_merges_url).content)
     logger.info("Done")
 
+    # Fetch and save pre-converted T5 text encoder model
```

Comment:
Shouldn't this code be run conditionally only when the mmdit model is specified and the T5 encoder is needed?

Reply:
It is optional (`--include-t5`) even when the model is an SD3/MMDiT variant